datascience/tables.py

```diff
@@ -2883,7 +2883,10 @@ class Row(tuple):
         _table = None  # Set by subclasses in Rows
 
         def __getattr__(self, column_label):
-            return self[self._table.column_index(column_label)]
+            try:
+                return self[self._table.column_index(column_label)]
+            except ValueError: #adding support for NumPy v1.18.0 as per changes in https://github.com/numpy/numpy/pull/14745
+                raise AttributeError("Attribute ({0}) not found in row.")
```

Comment:
Will the {0} work?  Is this missing a call to .format()?  Or does something magical happen with AttributeErrors that I'm unfamiliar with?

Reply:
🤦 nice catch. thanks!